src/components/AccountComponents/UserInfo.js

```diff
@@ -1,43 +1,58 @@
-import React, { Component, useState } from 'react'
-import UserService from '../../services/UserService'
-import Button from 'react-bootstrap/Button'
-import Collapse from 'react-bootstrap/Collapse'
+import React, { Component, useState } from 'react';
+import UserService from '../../services/UserService';
+import Button from 'react-bootstrap/Button';
+import Collapse from 'react-bootstrap/Collapse';
+import jwt_decode from 'jwt-decode';
+import { withRouter } from "react-router";
+import { Redirect } from 'react-router-dom';
 
-export default class UserInfo extends Component {
+class UserInfo extends Component {
   constructor(props) {
     super(props);
     this.state = {
         user: {},
         user_loaded: false,
+        redirect: false,
     }
   }
+  //loads in user
+  //keeps user fields hidden when the user cannot be loaded
   componentDidMount() {
     try{
```

Comment:
This makes a call to the database every time the page is reloaded. If the user is logged in it should be pulling the user info from storage.